bpf/bpf_lxc.c

```diff
@@ -305,7 +305,7 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 #ifdef HOST_IFINDEX
 			goto to_host;
 #else
-			return DROP_NO_LXC;
+			return DROP_HOST_UNREACHABLE;
```

Comment:
What's the point of this one? There are only two uses of the `DROP_NO_LXC` in the codebase AFAICT and this changes both of them to a new codepoint. Wouldn't it be easier to just change the error message in userspace?

Reply:
Good catch, DROP_NO_LXC was originally returned when no entry was found in the lxcmap for a dstIP that is part of the cluster range. These days, we punt everything to Linux instead.

However, because of this, we have a routing loop so I'm considering to add a lxcmap lookup in cilium_host to see if the srcIP is a known local endpoint and drop any ingress packet to cilium_host if a match is found as it represents a routing loop.

Reply:
@joestringer For this reason, it makes sense to keep the error code so I can use it for that purpose.